cpp/benchmarks/io/cuio_common.cpp

```diff
@@ -16,6 +16,8 @@
 
 #include <benchmarks/io/cuio_common.hpp>
 
+#include <cudf/detail/utilities/vector_factories.hpp>
```

Comment:
Is this include needed?

Reply:
Good catch. It should be a leftover.

Removed.